core/tests/test_utils.py

```diff
@@ -546,9 +592,11 @@ def _get_response(
         if expected_status_int >= 400:
             expect_errors = True
 
-        response = self.testapp.get(
-            url, params, expect_errors=expect_errors,
-            status=expected_status_int)
+        with self.swap(
```

Comment:
Explain why you are doing this, here and below.

Reply:
Done.